package.json

```diff
@@ -99,7 +101,10 @@
     "react-svg-inline": "2.1.0",
     "redux": "3.7.2",
     "redux-thunk": "2.2.0",
+    "sass-lint-fix": "1.12.1",
     "semver": "5.5.0",
+    "strip-ansi": "4.0.0",
+    "table-parser": "0.1.3",
```

Comment:
Why to you have to add `strip-ansi` and `table-parser` directly when they are deps of `ps-node` ?

Reply:
sass-lint-fix depends on these packages.
You can use: sass-lint --fix to autofix spacing issues as long as eslint --fix

Reply:
I still don't get it. if `sass-lint-fix` depends on them, npm will install them anyway.

Reply:
before u have asked to include all requirements

Reply:
If you mean "please, use exact versions of deps" from the comment above, I meant that we should use exact versions of deps in a semver sense - i.e. not using any kind of version specifiers that allow "automatic" version bumps, e.g. instead of `~1.0.3` or `^3.14.1` we should just use `1.0.3` and `3.14.1`